### DISCUSSION TITLE
The interaction effect in Ancombc model

### DISCUSSION THREAD

**Discussion by ADRS94** [FrederickHuangLin/ANCOMBC#225] [Q&A]
Hello!

I'm trying to perform an ANCOMBC analysis with 2 types of formulas:

`formula = Variable of interest + time + covariates `

and

`formula = Variable of interest * time + covariates `

I wan't to obtain the p_value of these interaction or something like de AIC or BIC to see if time has an interaction with my variable of interest.

Thanks !!

Adrià

**Comment by FrederickHuangLin** [comment]
Hello @ADRS94,

Regrettably, ANCOMBC does not currently support the inclusion of interactions. This is a focus area for our future development.

Best regards,
Huang

**Reply by ADRS94** [reply]
Thanks Frederick for your response.

I supose that i should perform an ANOVA for each taxa and do it manually for both formula.

Thanks and bests

Adrià

**Reply by timverlaan** [reply]
Hi Frederick,

I am just checking in to see if, as of july 2024, ANCOMBC is still not supporting interactions? I am looking to test the effect of obesity over each trimester of pregnancy on rectal and vaginal microbiome. I ran it using a premade interaction term (BMI:Trimester) (this ofcourse doesn't take into account the order of the Trimesters). Is it the case this will produce baised results? Do you have any suggestions how to use AMCOM-BC to test the effect of BMI changes across trimesters on differential abbundance of rectal and vaginal microbiome? 

stats.exchange post with more context: https://stats.stackexchange.com/q/649657/356093
